src/docker/mqtt/Dockerfile.mosquitto

```diff
@@ -0,0 +1,3 @@
+FROM eclipse-mosquitto:2.0.18-openssl
+
+COPY src/docker/mqtt/mosquitto.conf /mosquitto/config/mosquitto.conf
```

Comment:
So unfortunately this approach won't work for getting the config vars into mosquitto because it requires a volume mount, which is the whole issue we're trying to get around. As discussed on eng call

- [ ] Accept relevant config variables as environment variables inside `.example.env`
- [ ] Update `compose.dss-core.yaml` to pass the vars into the container
- [ ] Have container, on startup, create a config as for the processor Dockerfile like in `processor/start.sh`

Reply:
Noting this is resolved for *static* config loaded at build time